assets/scss/04-templates/_sidebars.scss

```diff
@@ -0,0 +1,6 @@
+.sidebar {
+  
+  h3 * {
+    font-size: 1.188rem;
```

Comment:
should this match other font-size vals --> 1.125rem?

Reply:
agreed with sienna, I think later we want to use variables for all font sizes in mayflower, so that sizing are consistent and people can change at the variable level

Reply:
this value was set in response to Christine's comment: "i think the H3s need to be revisited. they're a little intense in this sidebar." This is the size that was reviewed and approved.